privacy-protections/storage-blocking/helpers/commonTests.js

```diff
@@ -158,7 +158,7 @@ const commonTests = [
         },
         extra: () => {
             if (window.cookieStore) {
-                return cookieStore.get('cookiestoredata').then(cookie => {
+                return cookieStore.get('swcookiestoredata').then(cookie => {
```

Comment:
I'm confused why we have swcookiestoredata here and cookiestoredata the file below? Is this intended? They don't match up with their store/retrive functions now.

Reply:
Sorry, I got the cookie names mixed up. Now I've swapped them around.